packages/cli/src/zosfiles/edit/Edit.utils.ts

```diff
@@ -158,8 +159,8 @@ export class EditUtilities {
             lfFile.fileName,
             {
                 returnEtag: true,
-                binary: null,
-                encoding: null,
+                binary: lfFile.binary,
+                encoding: lfFile.encoding,
```

Comment:
Could you please add unit tests which expect that when the `localDownload` method is called with non-falsy values for `binary` or `encoding`, then those arguments get passed on to the `Download.ussFile` or `Download.dataSet` method?

It would probably be easiest to add this test to `Edit.utils.unit.test.ts` where the download APIs are already mocked.